certified-connectors/PDFco/apiDefinition.swagger.json

```diff
@@ -0,0 +1,727 @@
+                  "enum": ["Portrait", "Landscape"]
+                },
+                "printBackground": {
+                  "type": "boolean",
+                  "description": "enable/disable printing of background",
+                  "title": "printBackground",
+                  "enum": [true, false],
+                  "default": true
+                },
+                "header": {
+                  "type": "string",
+                  "description": "Set to HTML for header to be applied on every page at the header.",
+                  "title": "header",
+                  "format": "html",
+                  "x-ms-visibility": "advanced"
+                },
+                "footer": {
+                  "type": "string",
+                  "description": "Set to HTML for footer to be applied on every page at the bottom.",
+                  "title": "footer",
+                  "format": "html",
+                  "x-ms-visibility": "advanced"
+                },
+                "async": {
+                  "type": "boolean",
+                  "description": "",
+                  "title": "async",
+                  "enum": [true, false],
+                  "default": false,
+                  "x-ms-visibility": "advanced"
+                },
+                "encrypt": {
+                  "type": "boolean",
+                  "description": "",
+                  "title": "encrypt",
+                  "enum": [true, false],
+                  "x-ms-visibility": "internal"
+                },
+                "profiles": {
+                  "type": "object",
+                  "properties": {
+                    "CustomScript": {
+                      "type": "string",
+                      "description": "",
+                      "title": "profiles",
+                      "x-ms-visibility": "advanced"
+                    }
+                  },
+                  "description": "profiles"
+                }
+              },
+              "required": ["url"]
+            }
+          }
+        ]
+      }
+    },
+    "/v1/pdf/edit/add": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK, success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "",
+                  "title": "url",
+                  "x-ms-visibility": "important"
+                },
+                "pageCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "pageCount",
+                  "x-ms-visibility": "important"
+                },
+                "error": {
+                  "type": "string",
+                  "description": "",
+                  "title": "error",
+                  "enum": ["", "true", "false"],
```

Comment:
@hiren-byte could you please explain why do you have an empty string in possible values here?